exe/gold_miner

```diff
@@ -3,26 +3,32 @@
 $LOAD_PATH.unshift("#{__dir__}/../lib")
 
 require "gold_miner"
+require "dotenv"
 
-def debug(msg)
-  return if ENV["DEBUG"].nil?
+def load_env_file
+  Dotenv.load!
+  Dry::Monads::Success()
+rescue Errno::ENOENT
+  Dry::Monads::Failure("Could not load env file #{env_file.inspect}")
+end
 
-  puts "[DEBUG] #{msg}"
+def prepare
+  load_env_file
+    .bind { GoldMiner::Slack::Client.build(api_token: ENV["SLACK_API_TOKEN"]) }
+    .fmap { |slack_client|
+      GoldMiner::SlackExplorer.new(slack_client, GoldMiner::AuthorConfig.default)
+    }
```

Comment:
Maybe we could have a `SlackExplorer.build` method that tries to by the client first and uses default values so we could rewrite this in a single line?
```suggestion
    .bind { GoldMiner::SlackExplorer.build }
```